tools/Deployment/gulpfile.js

```diff
@@ -126,30 +114,6 @@ gulp.task("publish:azdevops-ppe", () => {
     return Myget.publishToMygetAsync(artifactsFolder, process.env.NUGETEXE, "anything", config.azdevops["ppeUrl"]);
```

Comment:
remove ?

Reply:
Why do we have Myget.publishToMyGetAsync here ?

Reply:
We need to publish to azdevops. I think the method name is confusing `publishToMygetAsync`... Let me update `myget.ts` to `nuget.ts`.